docs/api-docs/cart-and-checkout/checkout-sdk.md

```diff
@@ -53,6 +51,34 @@ Some payment methods, like PayPal, Amazon, etc., require you to provide addition
 ## Where can I get the SDK?
 The Checkout JS SDK and associated documentation is available from the [BigCommerce SDK Repo.](https://github.com/bigcommerce/checkout-sdk-js)
 
+## How can I update the SDK?
+The following steps describe how to update the Checkout SDK package in your existing custom checkout implementation.
+
+1. Fork and clone [checkout-js](https://github.com/bigcommerce/checkout-js), then install its dependencies with the clean slate install command.
+
+```shell title="Clone your fork and install dependencies"
+# Clone the repo
+git clone https://github.com/<your-user-name>/checkout-js checkout-js
+
+cd checkout-js
+
+# Install dependencies with a clean slate
+npm ci
+```
+
+2. Open your store's `package.json` file in your text editor.
```

Comment:
You can also go to the terminal and run `npm install @bigcommerce/checkout-sdk@<latest/version number>`